sql/core/src/main/scala/org/apache/spark/sql/streaming/ui/StreamingQueryTab.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.streaming.ui
+
+import org.apache.spark.internal.Logging
+import org.apache.spark.sql.execution.streaming.StreamQueryStore
+import org.apache.spark.ui.{SparkUI, SparkUITab}
+
+class StreamingQueryTab(val store: StreamQueryStore, sparkUI: SparkUI)
```

Comment:
Nit: Could we switch the tab position between "Structure Streaming" and "SQL".

Reply:
fixed